src/rez/utils/sourcecode.py

```diff
@@ -12,6 +14,8 @@
 import traceback
 import os.path
 
+_drive_start_regex = re.compile(r"^([A-Za-z]):\\")
```

Comment:
I don't think compiling the regex here has any performance benefits. It think it would be more beneficial if the regex was defined close to where it's used.